jobs/JRTMA3D_CLEAN

```diff
@@ -45,6 +45,19 @@ export HOMErtma3d=${HOMErtma3d:-$NWROOT/rtma.${rtma_ver}}
 # date and time for previous and next cycle
 export PDYHH="${PDY}${cyc}"
 
+# creating a sub-directory under top running directory to share the data among various steps
```

Comment:
Hi Gang, I am curious why you chose the "clean" job script to create your shared directory.  I would think you would want to create your shared directory in the GSI script and it looks like that is what you did.  So I am not sure if these changes to JRTMA3D_CLEAN are needed.

Reply:
Hi @AnnetteGibbs-NOAA,

Thank you for pointing this out. 

Yes, in current run, the "shared" directory is only needed after the GSI analysis, so I did create this shared directory in GSI script. Later I thought if this shared directory might be necessary for other steps, such as the prepfgs or prepobs, then it might be better to create the shared directory in the beginning of the cycle. "Clean" step seems to be the first step of each cycle in the workflow (in the xml file), so I also added the lines to create shared directory in Clean script.

Thanks,
-Gang